src/hotspot/share/interpreter/bootstrapInfo.cpp

```diff
@@ -229,8 +229,9 @@ void BootstrapInfo::print_msg_on(outputStream* st, const char* msg) {
   char what[20];
   st = st ? st : tty;
 
-  if (_indy_index != -1)
-    os::snprintf_checked(what, sizeof(what), "indy#%d", decode_indy_index());
+  if (_indy_index > -1) {
+    os::snprintf_checked(what, sizeof(what), "indy#%d", _indy_index);
+  }
```

Comment:
Since the `else` case doesn’t have braces, maybe omit the braces for this case as well?

Reply:
The if statements below use braces so I think it would be better to add braces to the else case.